### ISSUE TITLE
Android 8 & above conversion happens about 1/4th of the page only!!

### ISSUE THREAD

**Issue by vkreddyg** [dinesh4official/XFPDF#7]
hello,

its failing to convert entire page to pdf for Android 8 and above. I'm getting partial html page like, just top 5 lines of the html page and left of the page is blank. I see the html is fine but in Android 8 I see the Filestream size is about 1/4 of that of Filestream size from Android 5. So, i think its stopping the conversion at around 1/4 of the page.  Please help

Thanks
Vamshi

**Comment by dinesh4official** [comment]
@vkreddyg  can you share any sample HTML file?

**Comment by vkreddyg** [comment]
@dinesh4official I was able to fix it using below and it worked in both Android 5 & 8. Dont use Webview.LoadData, instead use LoadDataWithBaseURL.

//webpage.LoadData(_HtmlToPDF.HTMLString, "text/html; charset=utf-8", "UTF-8");
                webpage.LoadDataWithBaseURL(null, _HtmlToPDF.HTMLString, "text/html", "utf-8", null);

But, really your solution is amazing. Thanks for the work you did.. I had to change few things to show images in the pdf from html file. but other than that your solution is right on the spot..thanks
